sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationTests.cs

```diff
@@ -423,6 +420,7 @@ public async Task AnalyzeOperationWithMultipleActionsOfSameType()
         }
 
         [RecordedTest]
+        [ServiceVersion(Max = TextAnalyticsClientOptions.ServiceVersion.V3_2_Preview_2)]
```

Comment:
this is telling the tests, to run it in the Legacy side and not Language (which version is above 3.2-preview.2